cmd/utils/flags.go

```diff
@@ -1071,6 +1079,7 @@ func setBootstrapNodes(ctx *cli.Context, cfg *p2p.Config) {
 		urls = params.RinkebyBootnodes
 	case ctx.Bool(GoerliFlag.Name):
 		urls = params.GoerliBootnodes
+		// TODO(): BetaOPNetworkFlag default bootnodes
```

Comment:
Do we have an issue for this TODO somewhere?

Reply:
Thanks for the reminder. Opened CLI-4357 and will drop the TODO comment.